vocab/credentials/v2/vocabulary.yml

```diff
@@ -2,9 +2,9 @@ vocab:
   - id: cred
     value: https://w3.org/2018/credentials#
 
-prefix:
-  - id: odrl
-    value: http://www.w3.org/ns/odrl/2/
+# prefix:
+#   - id: odrl
```

Comment:
Prefer removing rather than commenting.

Reply:
I did not see any resolution on that so far, so I preferred to put it into a comment to be sure there is a consensus around this. Personally, I am in favor of removing it...